Tests/fixtures/smartphone-33.yml

```diff
@@ -4802,24 +4802,6 @@
     model: S62 Eco 3
   os_family: Android
   browser_family: Chrome
--
-  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; S554020) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.83 Mobile Safari/537.36
-  os:
-    name: Android
-    version: 8.1.0
-    platform: ""
-  client:
-    type: browser
-    name: Chrome Mobile
-    version: 83.0.4103.83
-    engine: Blink
-    engine_version: 83.0.4103.83
-  device:
-    type: tablet
-    brand: Fluo
-    model: L
-  os_family: Android
-  browser_family: Chrome
```

Comment:
https://olx.ba/artikal/43215858/fluo-l-1-16gb/
the smartphone. It would be great if we could fix it.